Source/Icebreaker/Services/MatchingService.cs

```diff
@@ -142,20 +152,21 @@ public async Task<int> MakePairsAndNotifyAsync()
         /// <param name="teamModel">DB team model info.</param>
         /// <param name="teamName">MS-Teams team name</param>
         /// <param name="pair">The pairup</param>
+        /// <param name="profiles">The pairup's respective profiles</param>
         /// <param name="cancellationToken">Propagates notification that operations should be canceled.</param>
         /// <returns>Number of users notified successfully</returns>
-        private async Task<int> NotifyPairAsync(TeamInstallInfo teamModel, string teamName, Tuple<ChannelAccount, ChannelAccount> pair, CancellationToken cancellationToken)
+        private async Task<int> NotifyPairAsync(TeamInstallInfo teamModel, string teamName, Tuple<ChannelAccount, ChannelAccount> pair, Tuple<string, string> profiles, CancellationToken cancellationToken)
```

Comment:
The parameter grouping here feels off. `Tuple<TeamsChannelAccount, string> person1, Tuple<TeamsChannelAccount, string> person2` seems more logical